docs/int/quickstart/alone/create-keys.md

```diff
@@ -20,7 +20,8 @@ Alternatively, the private key shares can be created in a lower-trust manner wit
 
 ## Pre-requisites
 
-- Ensure you have [docker](https://docs.docker.com/engine/install/) installed.
+- A basic [knowledge](https://docs.ethstaker.cc/ethstaker-knowledge-base/) of Ethereum nodes and validators.
+- Ensure you have [docker](https://docs.docker.com/desktop/install/ubuntu/#install-docker-desktop) installed.
```

Comment:
it doesn't have to be docker desktop, in fact most people will only have the engine installed and not the UI as they are not running a node with a UI framework installed